src/Tizen.Sensor/Tizen.Sensor/SensorEnumerations.cs

```diff
@@ -332,28 +332,34 @@ public enum DetectorState
     [EditorBrowsable(EditorBrowsableState.Never)]
     public enum AutoRotationState
     {
+        /// <summary>
+        /// Degree_Unknown state.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
```

Comment:
```suggestion
```
It should be removed, because it is not public api

Reply:
@myroot done!